shell/browser/net/network_context_service.cc

```diff
@@ -70,7 +70,11 @@ void NetworkContextService::ConfigureNetworkContextParams(
 
     network_context_params->file_paths =
         network::mojom::NetworkContextFilePaths::New();
-    network_context_params->file_paths->data_path = path;
+    network_context_params->file_paths->data_path =
+        path.Append(chrome::kNetworkDataDirname);
+    network_context_params->file_paths->unsandboxed_data_path = path;
+    network_context_params->file_paths->trigger_migration =
+        features::ShouldTriggerNetworkDataMigration();
```

Comment:
Might need to include `chrome/browser/browser_features.h`

Reply:
Oops forgot that we don't build the entire `//chrome/browser` target as part of https://github.com/electron/electron/blob/main/chromium_src/BUILD.gn#L13. Will need to add `//chrome/browser/browser_features.h, //chrome/browser/browser_features.cc` there.